crux-llvm/CHANGELOG.md

```diff
@@ -1,6 +1,27 @@
-# next
+# 0.6
+
+## New features
+
+* Improved support for translating LLVM debug metadata when the
+  `debug-intrinsics` option is enabled, including metadata that defines
+  metadata nodes after they are used.
+
+* Add overrides for certain floating-point operations such as `sin`, `cos`,
+  `tan`, etc. At the solver level, `crux-llvm` treats these as uninterpreted
+  functions, so `crux-llvm` is limited to reasoning about them up to basic,
+  syntactic equivalence checking.
+
+* Certain error messages now print the call stack of functions leading up to
+  the error.
+
+## Bug fixes
+
+* Make `--help` and `--version` respect the `--no-colors` flag.
+
+## Library changes
+
 * `LLVMConfig` now has an `indeterminateLoadBehavior` flag to control the
   `MemOptions` option of the same name. Refer to the `crucible-llvm` changelog
-  for more details.
+-  for more details.
```

Comment:
Intentional?
```suggestion
  for more details.
```

Reply:
Nope, definitely not. I think this is a leftover from a previous draft that I never fully reverted.